addons/storyshots/storyshots-core/src/frameworks/angular/helpers.ts

```diff
@@ -32,17 +32,32 @@ const createComponentFromTemplate = (template: string) => {
 };
 
 export const initModuleData = (storyObj: NgStory): any => {
-  const { component, template, props, moduleMetadata = {} } = storyObj;
+  const {
```

Comment:
What's this doing here?

Reply:
wtf

Reply:
I dunno man, this seems to have sorted itself